Index: pgpid/bin/pgpid-gen
===================================================================
--- pgpid/bin/pgpid-gen
+++ pgpid/bin/pgpid-gen
@@ -50,8 +50,8 @@ PGPI_VERSION="0.0.3"
 
 FACE_MARGIN_WIDTH="25/100"
 FACE_MARGIN_HEIGHT="50/100"
-TESSDATADIR="$(dirname "$0")/data/"
-GEOLIST_CENTROID="$(dirname "$0")/data/geolist_centroid.txt"
+TESSDATADIR="/usr/share/pgpid/data/
+GEOLIST_CENTROID="/usr/share/pgpid/data/geolist_centroid.txt"
 ONLYUDID=false
 
 ### Default option values ###
