--- a/bin/pgpid-gen
+++ b/bin/pgpid-gen
@@ -136,10 +136,10 @@
 
 ### functions ###
 
-. "$(dirname "$BASH_SOURCE")"/bl-log --no-act --log-level "$LOGLEVEL" --log-exit "$LOGEXITPRIO"
-. "$(dirname "$BASH_SOURCE")"/bl-interactive --
-. "$(dirname "$BASH_SOURCE")"/bl-security --
-. "$(dirname "$BASH_SOURCE")"/bl-json --
+. /usr/bin/bl-log --no-act --log-level "$LOGLEVEL" --log-exit "$LOGEXITPRIO"
+. /usr/bin/bl-interactive --
+. /usr/bin/bl-security --
+. /usr/bin/bl-json --
 
 # Implementation of ICAO doc 9303 part 3 (Specifications Common to all MRTDs - 4.9)
 icao9303_mrz_checkdigit() {
--- a/bin/pgpid-qrscan
+++ b/bin/pgpid-qrscan
@@ -107,9 +107,9 @@
 
 ### functions ###
 
-. "$(dirname "$BASH_SOURCE")"/bl-log --no-act --log-level "$LOGLEVEL" --log-exit "$LOGEXITPRIO"
-. "$(dirname "$BASH_SOURCE")"/bl-interactive --
-. "$(dirname "$BASH_SOURCE")"/bl-security --
+. /usr/bin/bl-log --no-act --log-level "$LOGLEVEL" --log-exit "$LOGEXITPRIO"
+. /usr/bin/bl-interactive --
+. /usr/bin/bl-security --
 
 # Do nothing else if sourced
 [[ "$BASH_SOURCE" == "$0" ]] || return 0
